crates/l2geth/src/rpc_client.rs

```diff
@@ -168,6 +168,40 @@ impl<T: Provider<Network>> ChunkInterpreter for RpcClient<'_, T> {
         self.handle
             .block_on(fetch_storage_node_async(&self.provider, node_hash))
     }
+
+    fn try_fetch_l1_msgs(&self, block_number: u64) -> Result<Vec<TxL1Message>> {
```

Comment:
This is only used in validium mode, right?

Reply:
Yes